### ISSUE TITLE
Lock version of axe-core to 2.5 to prevent accessibility test failures

### ISSUE THREAD

**Issue by kschuste** [cerner/terra-toolkit#38]
# Issue Description
With the 2.6.0 version of axe-core, the new accessibility rule for ensuring there is a main landmark in a document.  We are automatically pulling the latest release, 2.6.1 for axe-core.  We should lock it down to 2.5.0 until terra-core and other libraries are updated to handle the new release.

## Issue Type
- [ ] New Feature
- [ ] Enhancement
- [X] Bug
- [ ] Other

## Expected Behavior
Dependent library, terra-core, tests should not fail when running wdio tests.

## Current Behavior
When pulling in the 2.6 version of axe-core, wdio tests will fail for certain terra-core components.

**Comment by jmsv6d** [comment]
Closed with https://github.com/cerner/terra-toolkit/pull/39
